docs/logs.md

```diff
@@ -0,0 +1,55 @@
+# OpenTelemetry Rust Logs
+
+Status: **Work-In-Progress**
+
+## Introduction
+
+This document provides guidance on leveraging OpenTelemetry logs
+in Rust applications.
+
+## OTel Log Bridge API
+
+The OpenTelemetry Log Bridge API (part of the `opentelemetry` crate) is **not intended
+for direct use by application developers**. It is provided for authoring log
+appenders that bridge existing logging systems with OpenTelemetry. Bridges for
+`tracing` and `log` crates are already available.
+
+## Instrumentation Guidance
+
+1. **Use the `tracing` crate**: We strongly recommend using the
```

Comment:
Aware I sound like a broken record, but i'm still not sure about this, as using the tracing crate while The Great Tracing Bridge Integration Issue remains _can_ lead users down a frustrating path, depending on what they're doing.

Having said that, once Björns PR is merged, I think this is good guidance.